### ISSUE TITLE
Total Trip Distance Report showing extra 2021 row

### ISSUE THREAD

**Issue by SurbhiBakshi** [cityofaustin/atd-data-tech#6446]

**Comment by SurbhiBakshi** [comment]
![image.png](https://images.zenhubusercontent.com/5d5aebe557f95b00011f8e57/31415e36-d61d-40f7-973f-9d3a7f7b2ad6)



The report sourced from the Monthly Trips object shows an extra 2021 row with duplicate values. 
I added a false 2019 record see if there was a default rule that only pivoted more than 2-3 values, but that did not help, still see two 2021 rows. 
I downloaded the dataset as a CSV and pivoted it in Excel, and do not see an extra 2021 row, so don't think there are two variations in the formatting of 2021 (for example one with a space and one without), also if there were two formats, then the values corresponding to the 2021 rows would not have been identical.
